cmake/Modules/CocosConfigDefine.cmake

```diff
@@ -30,29 +30,27 @@ else()
      return()
  endif()
 
- # build mode, Debug is default value
- if(NOT CMAKE_BUILD_TYPE)
-     if(DEBUG_MODE)
-         set(CMAKE_BUILD_TYPE Debug)
-     else()
-         set(CMAKE_BUILD_TYPE Release)
-     endif()
- endif()
-
  if(CMAKE_GENERATOR)
```

Comment:
I think that CMAKE_GENERATOR has always some value.
It does not make sense to do if/else on CMAKE_GENERATOR

Reply:
thanks, the condition should check `CMAKE_CONFIGURATION_TYPES`.

now the logs like
```
-- CMAKE_GENERATOR: Unix Makefiles
-- CMAKE_BUILD_TYPE: Debug
```
or
```
-- CMAKE_GENERATOR: Xcode
-- CMAKE_CONFIGURATION_TYPES: Debug;Release
```